mq/plugins/cloudtrail.py

```diff
@@ -117,6 +118,10 @@ def onMessage(self, message, metadata):
         if not message['source'] == 'cloudtrail':
             return (message, metadata)
 
+        if 'requestparameters' in message['details']:
```

Comment:
A simpler way of writing these two lines would be to do something like this:
```py
if message.get('details', {}).get('requestparameters', {}).get('htmlpart') is not None:
```

Reply:
This isn't consistent with how the rest of our code is currently written, but I'll change it.

Reply:
I wouldn't consider this required; just a suggestion.